src/Service/Client.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace SilverStripe\CKANRegistry\Service;
+
+use GuzzleHttp\Psr7\Request;
+use RuntimeException;
+use SilverStripe\CKANRegistry\Model\Resource;
+use SilverStripe\Core\Extensible;
+use SilverStripe\Core\Injector\Injectable;
+
+class Client implements ClientInterface
```

Comment:
Is there a better name than `SilverStripe\CKANRegistry\Service\Client`? It feels a little too generic.

Reply:
I get that `CKANRegistry` is in the namespace but it still doesn't seem very clear that this is a base API client for CKAN specifically.

Reply:
How about APIClient? I originally had CKAN in there but it's also in the namespace so I removed it

Reply:
I'd be happy with `APIClient`